### PR TITLE
Cleaned variable names for targetting terminal class

### PR DESCRIPTION
**Pull request by gordon-cheung**

### CODE REVIEW

**Comment by bseto** [comment]
I like the new names better than the old ones! good :)

File: targeting_terminal.py [hunk 1]
```diff
@@ -16,22 +16,27 @@
...
-        self.fontSize_ = 15
-        self.maxTextWidth_ = 380
         self.terminal_rect_ = None
-        self.terminalHeight_ = 25
-        self.terminalRectX_ = 10
-        self.terminalRectY_ = 40
-        self.terminalWidth_ = 400
+        
+        # Terminal Visual Constants
+        self.terminal_RectX_ = 10
```
Comment (ExtraGravity) [inline targeting_terminal.py]: This is like a mix of capCase and under_scores. Should be consistent, so
`self.terminal_Rectx_` should be `self.terminal_rect_x_

**Comment by ExtraGravity** [comment]
They are better overall, the naming has improved, but small things like the consistent use of underscores needs to be improved.

**Comment by gordon-cheung** [comment]
Pliz Review again

**Comment by ExtraGravity** [comment]
Looks good, merge in from master first. Make sure everything is still the same, then we can merge in the pull request.

EDIT: Nevermind looks good, merging.

**Pull request merged**